### ISSUE TITLE
build-petalinux does not work for zedboard

### ISSUE THREAD

**Issue by ifakmme** [fpgadeveloper/ethernet-fmc-axi-eth#1]
Running the build-petalinux command for a zedboard image leads to the following errors during do_rootfs:

[log.do_rootfs.txt](https://github.com/fpgadeveloper/ethernet-fmc-axi-eth/files/4428696/log.do_rootfs.txt)

Is there anything I can do to get rid of these errors? I'm not even interested in these license files at all :)

Thanks,
Marco

**Comment by fpgadeveloper** [comment]
Hi @ifakmme 

Strangely I don't get these errors on my setup (Ubuntu 16.04 LTS, PetaLinux 2019.2). Can you let me know the details of your setup? You can download my bootfiles here:

https://opsero.com/downloads/boot/2019-2/zedboard_axieth-petalinux-2019-2.tar.gz

The only suggestion that I have is that you try to re-install PetaLinux and all of the dependencies.

Jeff

**Comment by ifakmme** [comment]
I tried using a docker container for petalinux, since 16.04 is a bit old: 
[Dockerfile.txt](https://github.com/fpgadeveloper/ethernet-fmc-axi-eth/files/4443999/Dockerfile.txt)

I've been using this container for petalinux for quite a while now.

I tried it in a virtual machine and the build works. Don't know whats wrong with the other setup, but since this works no need to investigate further.

Thanks!
Marco
